pybikes/data/ecobici_ba.json

```diff
@@ -0,0 +1,16 @@
+{
+    "instances": [
+        {
+            "tag": "ecobici-ba", 
```

Comment:
Tags, as possible, should be informative. Here I would go for `ecobici-buenos-aires`